index.bs

```diff
@@ -2094,6 +2159,40 @@ operation has no [=authenticator attachment modality=] selection option, so the
 credential|credentials=]. The [=client=] and user will then use whichever is available and convenient at the time.
 
 
+### Resident Key Requirement Enumeration (enum <dfn enum>ResidentKeyRequirement</dfn>) ### {#residentKeyRequirement}
+
+<xmp class="idl">
+    enum ResidentKeyRequirement {
+        "discouraged",
+        "preferred",
+        "required"
+    };
+</xmp>
+
+This enumeration's values describe the [=[RP]=]'s requirements for [=resident credentials=]:
+
+<div dfn-type="enum-value" dfn-for="ResidentKeyRequirement">
+    :   <dfn>discouraged</dfn>
+    ::  This value indicates the client, when communicating with an authenticator, SHOULD NOT instruct the authenticator
+        to create a [=resident credential=]. The authenticator SHOULD create a [=non-resident credential=], however either
+        a [=resident credential=] or [=non-resident credential=] will be accepted by the client.
+
+    :   <dfn>preferred</dfn>
+    ::  This value indicates the client, when communicating with an authenticator, SHOULD instruct the authenticator
+        to create a [=resident credential=]. If the authenticator is incapable of [=resident credentials=], 
+        the client MAY use the authenticator to create a [=non-resident credential=].
+
+    :   <dfn>required</dfn>
+    ::  This value indicates the client MUST use an authenticator capable of a [=resident credential=]. 
+        The authenticator MUST create a [=resident credential=] and MUST return an error if it cannot.
+</div>
+
+Note: It is intended for the client to use the [[#sctn-authenticator-credential-properties-extension]] in conjunction with the 
```

Comment:
s/intended for the client/intended for the [=[RP]=]/

Reply:
Done.